ToastNotification/ToastNotification.csproj

```diff
@@ -75,7 +75,8 @@ For some platforms, you may need to install platform-specific packages.
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Uno.UI" Version="4.5.12" />
```

Comment:
This line should be removed.

Uno.UI is simply a replacement of Windows APIs for non-UWP platforms. Therefore, a reference to Uno.UI for the target `uap10.0.15063` is unnecessary and may even potentially cause conflicts.

The `PackageReference` tag should instead be guarded with a `Condition=" '$(TargetFramework)' != 'uap10.0.15063' "`, see the code in the suggestion above.

```suggestion

```